packages/preset-mini/src/_theme/misc.ts

```diff
@@ -33,7 +33,7 @@ export const spacing = {
 
 export const duration = {
   DEFAULT: '150ms',
-  none: '0ms',
+  none: '0',
```

Comment:
```suggestion
  none: '0s',
```

0 is not a valid duration
![image](https://user-images.githubusercontent.com/19991745/216837434-332b43f9-5e7d-4a35-933b-8a8ba8c7de2a.png)